packages/uniswap/package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@rabbitholegg/questdk-plugin-utils": "workspace:*",
     "@rabbitholegg/questdk": "workspace:*",
-    "@uniswap/sdk-core": "^4.0.9",
-    "@uniswap/universal-router-sdk": "^2.0.2"
+    "@uniswap/sdk-core": "4.2.0",
+    "@uniswap/universal-router-sdk": "1.7.1"
```

Comment:
Is this from the weird situation where they like.... updated to an older version?

Reply:
![CleanShot 2024-03-11 at 14 03 22@2x](https://github.com/rabbitholegg/questdk-plugins/assets/62824345/64c4fc4c-14f3-40b3-ae03-e0d6c6453928)

I thought it was a bit weird. Not sure what the idea is behind the versioning 🤔